app/Http/Requests/CreateUserRequest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Validator;
+
+
+class CreateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|max:255',
+            'email' => 'required|email|unique:users',
+            'dob' => 'required|date',
+            'address'   => 'required',
+            'phone'   => 'required|numeric|unique:users',
+            'password'   => 'required|min:6'
+        ];
+    }
+    public function messages(){
+        return[
+            'name.required' => 'Trường không đươc bỏ trống',
```

Comment:
Tương tự như trên, em có thể set trong lang file, không cần set riêng trong từ request như thế này, có nhiều message mặc định của Laravel rồi, giờ muốn customize cái nào có thể đọc cách customize tại lnk https://laravel.com/docs/5.8/validation#custom-error-messages trong phần `Specifying Custom Messages In Language Files`

Reply:
Vậy còn validate **emaill.unique** thì vẫn phải cấu hình riêng cho nó chứ ạ

Reply:
E hiểu r ạ